specificatie/genereervariant/openapi.yaml

```diff
@@ -863,12 +836,16 @@ paths:
       - Adresseerbaar object
       summary: vindt verblijfsobjecten, ligplaatsen, standplaatsen
       description: Zoek actuele adresseerbare objecten (verblijfsobjecten, standplaatsen
```

Comment:
Als hiervan Mardown wordt gemaakt is het niet nodig om <br> te gebruiken voor een nieuwe regel